pages/polling/review.tsx

```diff
@@ -9,7 +9,7 @@ import invariant from 'tiny-invariant';
 import shallow from 'zustand/shallow';
 
 import { useBreakpointIndex } from '@theme-ui/match-media';
-import { isDefaultNetwork, getNetwork } from 'lib/maker';
+import { isDefaultNetwork, getNetwork, isTestnet } from 'lib/maker';
```

Comment:
Looks like `isTestnet` has been imported but isn't used. Should it be used in the fetching logic?